spec/components/schemas/message/visitor.ts

```diff
@@ -0,0 +1,23 @@
+import { SchemaObject } from 'openapi3-ts';
+import { createComponentRef } from '../../../../utils/ref';
+
+const visitor: SchemaObject = {
+  type: 'object',
```

Comment:
O visitor deveria ter title e description igual nossos outros schemas tem, não?

Reply:
Atualizado.

Reply:
Blz. Mas como é na terceira pessoa do singular, o certo é "indicates". E não tem sujeito oculto em inglês, então o certo é "It indicates". Se puder, arruma também o message-subscription e o direction que também tão errado.

Reply:
Atualizado